static/app/views/issueList/savedIssueSearches.tsx

```diff
@@ -165,4 +226,26 @@ const SavedSearchItemDescription = styled('div')`
   text-overflow: ellipsis;
 `;
 
+const OverflowMenu = styled(DropdownMenuControl)`
+  position: absolute;
```

Comment:
Is there a better way to position this? seems like absolute positioning is a bit weird for this?

Reply:
I don't think so - this is a button positioned over another button, and since HTML doesn't allow buttons within buttons, we have to position it like this

Reply:
ahh damn